tempodb/backend/raw.go

```diff
@@ -5,19 +5,27 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
+	"fmt"
 	"io"
 	"path"
 	"time"
 
+	"github.com/gogo/protobuf/proto"
 	"github.com/google/uuid"
 
 	tempo_io "github.com/grafana/tempo/pkg/io"
+	backend_v1 "github.com/grafana/tempo/tempodb/backend/v1"
 )
 
 const (
+	// JSON
 	MetaName          = "meta.json"
 	CompactedMetaName = "meta.compacted.json"
 	TenantIndexName   = "index.json.gz"
+
+	// Proto
+	TenantIndexNameProto = "index"
```

Comment:
The index is no longer gzipped.  I'm happy to add this back in if we'd like to see the file compressed.  Without it, in my small sample, the file itself was 2x.

Reply:
not gzipping would save a fair amount of memory/cpu. i think we'd need to see how large this file is uncompressed to decide.

note also that gzip'ing comes with the benefit of checksums added into the compressed file.

a potential option here is to use lz4, snappy, or zstd which all have better perf characteristics than gzip.

Reply:
What do you think about taking this change without the compression and then observing?  I think the approach of leaving the old indexes in place means we can do this again if we want to make the backend file `index.lz4` or whatever we want to do later.

Reply:
> What do you think about taking this change without the compression and then observing? 

i'm good if you're good :)